cli/tests/integration/object_store.rs

```diff
@@ -0,0 +1,29 @@
+use crate::common::{Test, TestResult};
+use hyper::{body::to_bytes, StatusCode};
+
+#[tokio::test(flavor = "multi_thread")]
+async fn object_store() -> TestResult {
+    const FASTLY_TOML: &str = r#"
```

Comment:
Can we add some negative checks for mis-formatted toml files, to make sure your nice error messages pop out?

Reply:
Added in e7555b0